infra.bs

```diff
@@ -119,6 +119,29 @@ will make the caller automatically rethrow the given value, if any, and thereby
 caller's algorithm. Using prose the caller has the ability to "catch" the exception and perform
 another action.
 
+<h4 id=algorithm-iteration>Iteration</h4>
+
+<p>There's a variety of ways to repeat a set of steps until a condition is reached. The Infra
+Standard is not (yet) exhaustive on this, but here are some examples:
+
+<ul>
+ <li><p>For each, as demonstrated later in this section.
```

Comment:
Editorially, you could insert links to for each with `, as defined <a lt="for each" for="list">for lists</a> and <a lt="for each" for="map">for maps</a>` if you wanted.

My perspective: it's somewhat important to define "for each" over data structures since it reinforces the fact that those data structures have an order. Other iterations are less important.